mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -70,7 +70,7 @@ abstract class PcCollector[T](
     case _ => rawPath
   def collect(
       parent: Option[Tree]
-  )(tree: Tree, pos: SourcePosition, symbol: Option[Symbol]): T
+  )(tree: Tree | EndMarker, pos: SourcePosition, symbol: Option[Symbol]): T
```

Comment:
I have it separately because it's not a subtype of `Tree` (from dotty compiler) but a special case class I created. End markers don't exist in abstract syntax trees, they are pruned since in a sense they provide no value there.

Reply:
Yes, thank you for the explanation! :+1: